cmd/explaintest/r/access_path_selection.result

```diff
@@ -0,0 +1,110 @@
+CREATE TABLE `pay_daily` (
+`id` varchar(50) NOT NULL DEFAULT '',
+`a_id` bigint(11) NOT NULL DEFAULT '0',
+`b_id` bigint(11) NOT NULL DEFAULT '0',
+`time_stamp` bigint(11) DEFAULT NULL,
+`date_key` int(20) DEFAULT NULL,
+`io_type` int(11) DEFAULT NULL,
+`update_source` int(11) DEFAULT NULL,
+`update_time` timestamp DEFAULT NULL,
+PRIMARY KEY (`id`, `a_id`, `b_id`),
+KEY `IDX_ID` (`a_id`, `b_id`, `time_stamp`),
+KEY `IDX_DK` (`date_key`),
+KEY `IDX_TENANT_POI_DATEKEY` (`a_id`, `b_id`, `date_key`),
+KEY `IDX_US` (`update_source`, `a_id`)
+);
+explain select id, a_id, b_id, time_stamp, date_key, update_source from pay_daily where a_id=12565 and b_id=159298363 and date_key=20180731;
+id	count	task	operator info
+IndexLookUp_18	0.00	root	
+├─IndexScan_16	0.00	cop	table:pay_daily, index:a_id, b_id, date_key, range:[12565 159298363 20180731,12565 159298363 20180731], keep order:false, stats:pseudo
+└─TableScan_17	0.00	cop	table:pay_daily, keep order:false, stats:pseudo
+explain select id from pay_daily where a_id=12565 and b_id=159298363 and time_stamp=10000;
+id	count	task	operator info
+Projection_4	0.00	root	test.pay_daily.id
+└─IndexLookUp_10	0.00	root	
+  ├─IndexScan_8	0.00	cop	table:pay_daily, index:a_id, b_id, time_stamp, range:[12565 159298363 10000,12565 159298363 10000], keep order:false, stats:pseudo
+  └─TableScan_9	0.00	cop	table:pay_daily, keep order:false, stats:pseudo
+explain select id from pay_daily where update_source>12565 and a_id<159298363;
```

Comment:
How can we decide if `IDX_US` is better or `IDX_ID`/`IDX_TENANT_POI_DATEKEY ` is better for this query when we don't have stats info?

Reply:
This unclear case has been removed now.